### PR TITLE
add more i18n keys

### PR DESCRIPTION
**Pull request by ann-kilzer**
## Part of #33 

More i18n keys. The en.json file was updated here: https://github.com/ourjapanlife/findadoc-localization/pull/42

To test: Confirm that various languages work on the netlify environment.

COVID-19_Vaccination_Info

### CODE REVIEW

File: components/CancellationList.vue [hunk 1]
```diff
@@ -63,16 +66,18 @@
               text
               @click="sendReport"
             >
-              Report
+              {{ $t("cancelList.report.submitReport") }}
+            </v-btn>
+            <v-btn color="secondary" text @click="cancelReport">
```
Comment (ann-kilzer) [inline components/CancellationList.vue:71]: In general, cancel buttons should be secondary as they are not the primary action

https://uxplanet.org/primary-secondary-action-buttons-c16df9b36150

**Comment by ann-kilzer** [comment]
I ran prettier in here and apparently there are a lot of diffs. I will configure that as a github action so we don't have so many small diffs on future PRs

File: components/CancellationList.vue (hunk 1 above)
Comment (Ermish) [reply components/CancellationList.vue:71]: +1

**Review by Ermish — APPROVED** [review]
LFTM

**Pull request merged**